node/service/src/tests.rs

```diff
@@ -521,7 +508,7 @@ fn chain_0() -> CaseVars {
 
 	let a1 = builder.fast_forward_approved(0xA0, head, 1);
 	let a2 = builder.fast_forward_approved(0xA0, a1, 2);
-	let a3 = builder.fast_forward_approved(0xA0, a2, 3);
+	let a3 = builder.fast_forward_disputed(0xA0, a2, 3);
```

Comment:
If 0xA3 is not disputed, then it is automatically `undisputed_chain`, since `chain.disputes.is_empty() == true`. However, this is contrary to the expectation that 0xA2 is the `undisputed_chain`.

Reply:
Same as before, I'd prefer to keep the test case input the same as before, if there is a need for additional ones, C&P :)